miqa/core/tests/factories.py

```diff
@@ -10,3 +14,75 @@ class Meta:
     email = factory.Faker('safe_email')
     first_name = factory.Faker('first_name')
     last_name = factory.Faker('last_name')
+
+
+class SiteFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Site
+
+    id = factory.Faker('uuid4')
+    name = factory.Faker('safe_email')
+
+    creator = factory.SubFactory(UserFactory)
+
+
+class SessionFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Session
+
+    id = factory.Faker('uuid4')
+    name = factory.Faker('safe_email')
+
+    creator = factory.SubFactory(UserFactory)
+
+    import_path = '/fake/path'
+    export_path = '/fake/path'
+
+
+class ExperimentFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Experiment
+
+    id = factory.Faker('uuid4')
+    name = factory.Faker('safe_email')
+    note = factory.Faker('pystr')
+
+    session = factory.SubFactory(SessionFactory)
+
+
+class ScanFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = Scan
+
+    id = factory.Faker('uuid4')
+
+    experiment = factory.SubFactory(ExperimentFactory)
+    site = factory.SubFactory(SiteFactory)
+
+    scan_id = factory.Faker('pystr')
+    scan_type = factory.Faker('pystr')
+    decision = 'Good'
```

Comment:
Decisions are stored in the DB in all caps
```suggestion
    decision = 'GOOD'
```